suite-native/receive/src/components/ShowAddressButtons.tsx

```diff
@@ -33,6 +61,45 @@ export const ShowAddressButtons = ({ onShowAddress }: ShowAddressButtonsProps) =
             <TextButton size="small" onPress={handleOpenEduLink} iconRight="arrowUpRight">
                 <Translation id="moduleReceive.receiveAddressCard.showAddress.learnMore" />
             </TextButton>
+            <BottomSheet
```

Comment:
What about moving this BottomSheet for a separate file?

Reply:
[a304a70](https://github.com/trezor/trezor-suite/pull/11928/commits/a304a701d2b4c6d9f09a92db7949c21bdb0e0571)